drivers/telescope/rainbow.h

```diff
@@ -160,6 +176,10 @@ class Rainbow : public INDI::Telescope, public INDI::GuiderInterface
         const std::string getSlewErrorString(uint8_t code);
         uint8_t m_SlewErrorCode {0};
 
+        ISwitchVectorProperty SaveAlignBeforeSyncSP;
+        ISwitch SaveAlignBeforeSyncS[1];
```

Comment:
I think it's better to make this SaveAlignBeforeSyncS[2] and use INDI_ENABLED and INDI_DISABLED switches. I understand you want to use it as a toggle (which works), but using two switches (Enabled and Disabled) is more *clear* to the end user.

Finally, try to add this switch to the saveConfig(...).

Reply:
Yes definitely better to align to the common UX, I've updated this also